app/code/Magento/Quote/Model/ResourceModel/Quote/Item/Collection.php

```diff
@@ -61,6 +67,7 @@ public function __construct(
         \Magento\Quote\Model\ResourceModel\Quote\Item\Option\CollectionFactory $itemOptionCollectionFactory,
         \Magento\Catalog\Model\ResourceModel\Product\CollectionFactory $productCollectionFactory,
         \Magento\Quote\Model\Quote\Config $quoteConfig,
+        \Magento\Store\Model\StoreManagerInterface $storeManager = null,
```

Comment:
Hi @mage2pratik, thanks for collaboration. The optional parameter should be passed last in the params list, currently, it's [backward-incompatible changes](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)